flashcard_runner_it4.rb

```diff
@@ -0,0 +1,13 @@
+require './lib/card'
+require './lib/turn'
+require './lib/deck'
+require './lib/round'
+require './lib/card_generator'
+
+cards = CardGenerator.new('.lib/cards.txt').cards
+deck = Deck.new(cards)
+round = Round.new(deck)
+
+round.start
```

Comment:
This, as well as the CardGenerator class's pseudocode is a great start to Iteration 4. I liked seeing you write out comments for what you knew you'd need to do in that method.